src/librustc_driver/driver.rs

```diff
@@ -1031,6 +1031,7 @@ pub fn phase_4_translate_to_llvm<'a, 'tcx>(tcx: TyCtxt<'a, 'tcx, 'tcx>,
         passes.push_pass(box mir::transform::simplify::SimplifyCfg::new("elaborate-drops"));
 
         // No lifetime analysis based on borrowing can be done from here on out.
+        passes.push_pass(box mir::transform::inline::Inline);
```

Comment:
That comment is about line 1031, i.e. nothing below the comment can invalidate it.